docs/data-sources/gateway_control_plane_list.md

```diff
@@ -0,0 +1,151 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "konnect_gateway_control_plane_list Data Source - terraform-provider-konnect"
+subcategory: ""
+description: |-
+  GatewayControlPlaneList DataSource
+---
+
+# konnect_gateway_control_plane_list (Data Source)
+
+GatewayControlPlaneList DataSource
+
+## Example Usage
+
+```terraform
+data "konnect_gateway_control_plane_list" "my_gatewaycontrolplanelist" {
+  filter = {
+    cloud_gateway = true
+    cluster_type = {
+      eq  = "...my_eq..."
+      neq = "...my_neq..."
+      oeq = "...my_oeq..."
+    }
+    id = {
+      eq  = "...my_eq..."
+      oeq = "...my_oeq..."
+    }
+    name = {
+      contains = "...my_contains..."
+      eq       = "...my_eq..."
+      neq      = "...my_neq..."
+      oeq      = "...my_oeq..."
+    }
+  }
+  labels      = "key:value,existCheck"
+  page_number = 1
+  page_size   = 10
+  sort        = "name desc"
```

Comment:
The API endpoint seems to only supports sorting on `created_at` - https://kongstrong.slack.com/archives/CQK8J4VN3/p1744182256786029
Should we update the example?

Reply:
Good catch. Yes please.

This is a shared component. It is safe to update the shared component to use created at as all entities support this field